django-stubs/conf/global_settings.pyi

```diff
@@ -541,3 +541,9 @@ SECURE_REDIRECT_EXEMPT: list[str]
 SECURE_REFERRER_POLICY: str
 SECURE_SSL_HOST: str | None
 SECURE_SSL_REDIRECT: bool
+
+##################
+# CSP MIDDLEWARE #
+##################
+SECURE_CSP: dict[str, Any] = {}
+SECURE_CSP_REPORT_ONLY: dict[str, Any] = {}
```

Comment:
```suggestion
SECURE_CSP: dict[str, Any]
SECURE_CSP_REPORT_ONLY: dict[str, Any]
```

Reply:
We might also type it as `Sequence[str]` instead of `Any` maybe?

Reply:
Apparently `SECURE_CSP_REPORT_ONLY = {"report-uri": "/path/to/reports-endpoint/"}` is a valid value, so it's not always `Sequence`.